rdrf/rdrf/models/definition/models.py

```diff
@@ -2017,11 +2016,20 @@ def _generate_input_form_class(self, inputs):
         def create_field(input_spec):
             field_type = input_spec["field_type"]
             label = input_spec["label"]
+            kwargs = {}
+            widget = None
             if field_type == "date":
-                klass = forms.DateField
+                from django import forms
+                from rdrf.forms.dynamic.fields import IsoDateField
+                klass = IsoDateField
+                widget = forms.DateInput(attrs={'class': 'datepicker'},
+                                         format='%dd-%mm-%YY')
+                kwargs["widget"] = widget
+                kwargs["input_formats"] = ["%d-%m-%Y"]
             else:
-                raise NotImplementedError("don't support yet")
-            return klass(label=label)
+                raise NotImplementedError("don't non date fields yet")
```

Comment:
you meant "not supported" but missed the word

Reply:
ok